lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ServiceAndContextConfigurationProvider.cs

```diff
@@ -36,31 +34,30 @@ namespace Org.Apache.REEF.IMRU.OnREEF.Driver
     /// </summary>
     /// <typeparam name="TMapInput"></typeparam>
     /// <typeparam name="TMapOutput"></typeparam>
-    internal sealed class ServiceAndContextConfigurationProvider<TMapInput, TMapOutput>
+    /// <typeparam name="TDataHandler"></typeparam>
+    internal sealed class ServiceAndContextConfigurationProvider<TMapInput, TMapOutput, TDataHandler>
     {
-        private static readonly Logger Logger = Logger.GetLogger(typeof(ServiceAndContextConfigurationProvider<TMapInput, TMapOutput>));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(ServiceAndContextConfigurationProvider<TMapInput, TMapOutput, TDataHandler>));
 
-        private readonly Dictionary<string, ContextAndServiceConfiguration> _configurationProvider;
-        private readonly ISet<string> _failedEvaluators;
-        private readonly ISet<string> _submittedEvaluators; 
+        private readonly Dictionary<string, string> _configurationProvider;
```

Comment:
Why is this named `_configurationProvider`? It doesn't seem like it provides any configurations.